package.json

```diff
@@ -22,8 +22,8 @@
     "package:chrome": "zip -r web-ext-artifacts/chrome_relay-${npm_package_version}.zip ./src",
     "test": "echo \"Error: no test specified\" && exit 1",
     "restore-locales-github": "cd src/_locales && git restore .github/workflows/update-upstream-relay-repo.yml",
-    "remove-locales-github": "rm -r src/_locales/.github",
-    "web-ext-run": "rm -r src/_locales/.github && web-ext run -s src/",
+    "remove-locales-github": "rm -rf src/_locales/.github",
```

Comment:
The `-f` suppresses the warning that the file has been removed when you run the `web-ext-run` command more than once locally. Added back. 

cc/ @Vinnl